src/test/java/org/asciidoctor/maven/process/SourceDocumentFinderTest.java

```diff
@@ -1,21 +1,22 @@
 package org.asciidoctor.maven.process;
 
-import org.junit.jupiter.api.Test;
-
 import java.io.File;
+import java.nio.file.Paths;
 import java.util.Collections;
 import java.util.List;
 
+import org.junit.jupiter.api.Test;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class CustomExtensionDirectoryWalkerTest {
+public class SourceDocumentFinderTest {
```

Comment:
Previous tests trusted the underlying implementation and did focus on covering files/folders combinations. But if we replace it with custom implementation we should add more tests, for example, testing with empty and multiple extensions in the second param, non existing paths, etc.